swagger.yml

```diff
@@ -891,42 +891,63 @@ components:
     Alert:
       title: Alert
       type: object
+      x-examples:
+        example-1:
+          AlertAffectedTechnology: null
+          AlertCategory: AGENT
+          AlertCode: NO_DATA
+          AlertSeverity: CRITICAL
+          AlertStatus: NEW
+          Date: '2020-07-08T06:15:24.391Z'
+          Description: No data received from the host publicitate-36d06ca83eafa454423d2097f4965517 in the last 0 days
+          Hostname: publicitate-36d06ca83eafa454423d2097f4965517
+          OtherInfo:
+            Hostname: publicitate-36d06ca83eafa454423d2097f4965517
+          _id: 5f05647c682a867d4bdb28a1
       properties:
         OtherInfo:
           type: object
           properties:
             DbName:
               type: string
+              description: The name of the database
             Hostname:
               type: string
+              description: The name of the host
         Hostname:
           type: string
+          description: The name of the host(probably a bug)
         Description:
           type: string
+          description: 'The message alert receive from the host at that time '
         AlertCategory:
           type: string
           enum:
             - SYSTEM
             - AGENT
             - LICENSE
+          description: 'contains: "ENGINE", "AGENT", "LICENSE"'
         AlertAffectedTechnology:
           nullable: true
           type: string
           enum:
             - Oracle/Database
             - Oracle/Exadata
+          description: '"NULL" for now '
         AlertStatus:
           type: string
           enum:
             - NEW
             - ACK
           example: NEW
+          description: '"NEW", "ACK"'
```

Comment:
Is the description redundant to the enum property?

Reply:
I have removed the description. The enum was not visible from the StopLight view